articles/libraries/lock/index.md

```diff
@@ -120,21 +120,13 @@ This is just one example of how **Lock 10** could work with a **Single Page Appl
 
 The [API reference][lock-api] provides more comprehensive documentation of the API and its various available methods, and the [Customization reference][lock-customization] details out the different options that you have for customizing Lock to meet your specific needs.
 
-## Examples
-
-The **example** directory has a ready-to-go app. In order to run it you need [node](http://nodejs.org/) installed.
-
-Then execute `npm i` to install dependencies (only once) and `npm example` from the root of this project.
-
-Finally, point your browser at `http://localhost:3000/` and play around.
-
-## Start using Lock!
+## Using auth0.js
 
-${lockSDK}
+There is no need to use `getClient()` any more - instead, you can just instantiate a new `Auth0` object.
 
-## Using auth0.js
+To get `Auth0` with Lock, if you used the CDN or bower for installation, you merely have to instantiate your object (see below). If you are retrieving Lock via npm, you will have to include `auth0.js` in your project's dependencies and import it.
```

Comment:
- "if you include the script from our CDN or installed Lock from bower"
- "if you installed Lock from npm"
- The npm package name is `auth0-js`